orttraining/orttraining/python/training/training_session.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, path_or_bytes, parameters, sess_options=None):
         self._inputs_meta = self._sess.inputs_meta
         self._outputs_meta = self._sess.outputs_meta
 
-        Session.__init__(self, self._sess)
+        Session.__init__(self)
```

Comment:
No point passing self._sess through as it's already set in this class (so Session can see it), and Session.__init__ wasn't doing anything with it anyway.